src/foam/core/Static.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * @license
+ * Copyright 2017 The FOAM Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+foam.CLASS({
+  package: 'foam.core',
+  name: 'Static',
+  extends: 'foam.core.AbstractMethod',
+
+  documentation: 'An Axiom for defining static methods.',
+
+  properties: [
+    //get prop from foam.core.AbstractMethod
+  ],
+
+  methods: [
+    function exportAs(obj) {

```

Comment:
Needs to installInClass() and installInProto() (like in Constant.js) so that these are added to the JS class.